index.js

```diff
@@ -1,3 +1,14 @@
+if(!process.env.now) require('dotenv').config();
```

Comment:
I don't think this is doing what you think it is doing. an if statement needs to be contained in a scope

`if ( checkThisCondition ) {if TRUE -> go in this scope}`